src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1293,8 +1293,8 @@ def forward(
             return (pred_logits, image_class_embeds)
 
         # Normalize image and text features
-        image_class_embeds /= torch.linalg.norm(image_class_embeds, dim=-1, keepdim=True) + 1e-6
-        query_embeds /= torch.linalg.norm(query_embeds, dim=-1, keepdim=True) + 1e-6
+        image_class_embeds = image_class_embeds / torch.linalg.norm(image_class_embeds, dim=-1, keepdim=True) + 1e-6
```

Comment:
wait. I think we should do

(torch.linalg.norm(image_class_embeds, dim=-1, keepdim=True) + 1e-6) instead

Reply:
```suggestion
        image_class_embeds = image_class_embeds / (torch.linalg.norm(image_class_embeds, dim=-1, keepdim=True) + 1e-6)
```